hail/src/test/scala/is/hail/io/compress/BGzipCodecSuite.scala

```diff
@@ -114,4 +115,57 @@ class BGzipCodecSuite extends SparkSuite {
     }
     p.check()
   }
+
+  @Test def testVirtualSeek() {
+    sc.hadoopConfiguration.setLong("mapreduce.input.fileinputformat.split.minsize", 1L)
+    // real offsets of the start of each block
+    val blockStarts = Array[Long](14653, 28034, 40231, 55611, 69140, 82949, 96438, 110192, 121461, 133703,
+      146664, 158711, 171239, 181362)
+    // offsets into the uncompressed file
+    val uncompBlockStarts = Array[Long](65280, 130560, 195840, 261120, 326400, 391680, 456960, 522240,
+      587520, 652800, 718080, 783360, 848640, 913920)
```

Comment:
These are just a random subset of blocks?

Reply:
These are all the blocks for that particular file.

Reply:
Why doesn't the first one start at 0?

Reply:
Forgot that one. I've added it since.

Reply:
:+1: